CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to TypedStruct
+# Contributing
```

Comment:
What’s the point for this change? This title follows my own code convention, so if I change it here I’ll need to update all my projects to be consistent. I can accept such change if justified.

Reply:
> What’s the point for this change? This title follows my own code convention, so if I change it here I’ll need to update all my projects to be consistent. I can accept such change if justified.

Removed because I want to shorten to title in TOC. I've reverted this change as I've figured out how to rename the added page title in `mix.exs`.